README.md

```diff
@@ -72,12 +72,14 @@ This will create an OpenTelemetry Collector instance named `simplest`, exposing
 
 The `config` node holds the `YAML` that should be passed down as-is to the underlying OpenTelemetry Collector instances. Refer to the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) documentation for a reference of the possible entries.
 
-> 🚨 **NOTE:** At this point, the Operator does _not_ validate the contents of the configuration file: if the configuration is invalid, the instance will still be created but the underlying OpenTelemetry Collector might crash.
```

Comment:
this is still mostly true, why are we removing it?

Reply:
(right now we validate the base structure, but not much more)

Reply:
Well, we are validating that ports are integers (since before the changes introduced by this PR). This note is misleading. Only a few lines below it we say that there's some validation. This might leave users confused about which statement is true.